CONTRIBUTING.md

```diff
@@ -29,7 +29,13 @@ If you don't know what contribution you can work on, here are a few suggestions:
 * Write documentation or fix the existing documentation
 * If you still don't know what could be a good task for you, do not hesitate to contact us.
 
-# You should open a FIP if
+# Fish Improvement Proposal (FIP)
+
+The purpose of a FIP is to explain an improvement you would like to make to Iron Fish, and get consensus from the core development team of Iron Fish. The reason is that we want to make sure your change fits inside of the product vision, and that you don't try to fix a bug in a piece of code already being refactored.
+
+You can submit a FIP by filing an issue here, https://github.com/iron-fish/ironfish/issues/new/choose
```

Comment:
You can inline the link or use a colon. In any case, it looks good to me.

Reply:
fixed